main.go

```diff
@@ -293,6 +293,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	serviceReconciler := controllers.NewActiveMQArtemisServiceReconciler(
```

Comment:
Is the one for ActiveMQArtemisApp missing?

Reply:
indeed it is, for the deploy operator scenario to work!